.github/CODEOWNERS

```diff
@@ -7,8 +7,17 @@
 # review when someone opens a pull request.
 *       @aws/dl-containers
 
+
+# Common files that is managed by multiple owners
+available_images.md @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers
+release_images_inference.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers
+release_images_training.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers
+.release_images_template.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers
+dlc_developer_config.toml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-dgl-reviewers @aws/dlc-trcomp-reviewers @aws/dlc-smdebug-reviewers
```

Comment:
I think this one we will probably still want dl-containers to own. Once the PR reviewer reverts the toml, dl-containers should no longer be required

Reply:
what if there is a new configuration added to the file?

Reply:
the above will also need changes to src/ files which dlc-containers should review. Will remove the file.